packages/admin-stories/src/docs/components/Table/FilterBar/stories/Default.stories.tsx

```diff
@@ -0,0 +1,207 @@
+import {
+    Field,
+    FilterBar,
+    FilterBarMoreFilters,
+    FilterBarPopoverFilter,
+    FinalFormInput,
+    FinalFormRangeInput,
+    FinalFormSearchTextField,
+    FinalFormSwitch,
+    Table,
+    TableFilterFinalForm,
+    useTableQueryFilter,
+} from "@comet/admin";
+import { FinalFormReactSelectStaticOptions } from "@comet/admin-react-select";
+import { Box, Divider, FormControlLabel, Typography } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import faker from "faker";
+import * as React from "react";
+
+interface ColorFilterFieldProps {
+    colors: string[];
+}
+
+const ColorFilterField: React.FC<ColorFilterFieldProps> = ({ colors }) => {
+    const options = colors
+        .filter((color, index, colorsArray) => colorsArray.indexOf(color) == index) //filter colorsArray to only have unique values as select options
+        .map((color) => {
+            return { value: color, label: color };
+        });
+
+    return <Field name="color" type="text" component={FinalFormReactSelectStaticOptions} fullWidth options={options} />;
+};
+
+interface IFilterValues {
+    brand: string;
+    model: string;
+    color: string;
+    horsepower: {
+        min: number;
+        max: number;
+    };
+    price: {
+        min: number;
+        max: number;
+    };
+    owner: {
+        firstname: string;
+        lastname: string;
+    };
+    query: string;
+}
+
+interface IExampleRow {
+    id: number;
+    model: string;
+    brand: string;
+    color: string;
+    horsepower: number;
+    price: string;
+    owner: {
+        firstname: string;
+        lastname: string;
+    };
+}
+
+export const randomTableData = Array.from(Array(15).keys()).map((i): IExampleRow => {
+    return {
+        id: i,
+        model: faker.vehicle.model(),
+        brand: faker.vehicle.manufacturer(),
+        color: faker.commerce.color(),
+        horsepower: faker.datatype.number({ min: 50, max: 200 }),
+        price: faker.commerce.price(100, 1000, 2),
+        owner: {
+            firstname: faker.name.firstName(),
+            lastname: faker.name.lastName(),
+        },
+    };
+});
+
+storiesOf("stories/components/Table/Filterbar/Filterbar/Default", module).add("Default", () => {
```

Comment:
In my opinion, 120 LOC for the default story of a component is too long

Reply:
i know it's long but thats needed to show how it works with a table i think.

Reply:
I would suggest to:

- create a basic example where you show the usage of FilterBar together with a table (1-2 filters are enough, no More button, no range filter just basic filter)
- show advanced features as their own example, this without full table integration (as you have already done for 2 features)

Reply:
Now the stories are pretty neat 👏🏼